packages/core/core.ts

```diff
@@ -135,11 +137,10 @@ function cleanStack(stack: string): string {
 	return result.join("\n");
 }
 
-function importModule(name: string) {
-	// eslint-disable-next-line @typescript-eslint/no-var-requires
-	return require(name);
+async function importModule(name: string) {
```

Comment:
What do you think about returning a `FuzzTarget` type here? This would get rid of the implicitly returned `any` and  document the code. Like: 
```typescript
interface FuzzTarget {
  [fuzzFunction: string]: fuzzer.FuzzFn;
}
```

Reply:
As we're currently calling `importModule` for the `customHooks` as well it feels off to returns such an interface

Reply:
Ah, right, I didn't notice the other places where `importModule` is used. Reverting would be an option.

True, there is a naming conflict. We could argue that one is a property and the other a type, so it would be fine. But I would tend to rename the `fuzzTarget` property in  the options interface, if we stick to the type. Whichever way you choose.